test/external/external_test_yolov8.py

```diff
@@ -46,7 +46,7 @@ def test_forward_pass_torch_onnx(self):
 
```

Comment:
You did not change these, but it might make sense defince `weights_folder`. `Path(__file__).parent.parent.parent` can also be `Path(__file__).parents[3]`